test/org/netbeans/jemmy/operators/JInternalFrameOperatorCloseTest.java

```diff
@@ -96,11 +97,7 @@ public void internalFrameDeactivated(InternalFrameEvent e) {
 
             @Override
             public void internalFrameClosing(InternalFrameEvent e) {
-                try {
-                    this.wait(80000);
-                } catch (InterruptedException e1) {
-                    e1.printStackTrace();
-                }
+                    new QueueTool().waitEmpty();
```

Comment:
There is a comment below, which explains what should be here:
// Making not to close the fame for 1 minute and expecting TimeoutExpiredException
// from waitClosed()

Which would mean that the lines in internalFrameClosing(InternalFrameEvent e) should just delay the execution for  minute. The reason for doing that, it looks like, is that there is no other way to detect window closing but by the timeout.

Your fix is doing something completely different.

Reply:
Removed negative test scenario. 
Currently, Testing that Internal frame is closing or not.